src/components/SearchBox.tsx

```diff
@@ -1,61 +1,94 @@
+//simple imports
 import { useState } from 'react';
-import Chip from '@mui/material/Chip';
 import Autocomplete from '@mui/material/Autocomplete';
 import TextField from '@mui/material/TextField';
 import Stack from '@mui/material/Stack';
-import { debounce } from 'lodash';
 import axios from 'axios'
 import Button from '@mui/material/Button';
-
-
+import ComparisonTable from './ComparisonTable';
+import { Typography } from '@mui/material';
+import { useDebouncedCallback } from 'use-debounce';
+import Alert from '@mui/material/Alert';
+import AlertTitle from '@mui/material/AlertTitle'; 
+import Snackbar from '@mui/material/Snackbar';
 
 export default function Tags() {
   //variable initializations
     const [num,setNum]=useState(0)
     const [inputValue, setInputValue] = useState('');
-    const [searchResults,setSearchResults]=useState([])
-
-
-
+    const [searchResults,setSearchResults]=useState([]);
+    const [selectedPacakges, setSelectedPackages]=useState([]);
+    const [comparisonCheck, setComparisonCheck] = useState(false);
+    const [open, setOpen] = useState(false);
```

Comment:
The name of all the states should be readable. 
For example open,setOpen is not specifying what is opening or closing.